Source/palette.cpp

```diff
@@ -212,7 +212,7 @@ void DecreaseGamma()
 
 int UpdateGamma(int gamma)
 {
-	if (gamma) {
+	if (gamma != 0) {

```

Comment:
```suggestion
	if (gamma > 0) {
```
This function is called with gamma having the value 0 or something in the range [30, 100]